packages/vue/src/stories/releases/v0.10.9/v0.10.9.stories.mdx

```diff
@@ -0,0 +1,11 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Releases/v0.10.9 - Latest/Change log" />
+
+# v0.10.9
+
+## 🚀 Features
+
+- SfQuantitySelector: Add min and max props to set minimum or maximum allowed QTY
```

Comment:
For a change in props you could use a table in a similar way we used it in the previous release note but you can still leave this note here

```
## ✨ Props

<table>
  <thead>
    <tr>
      <th colspan="5">Props</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Component</td>
      <td>Before</td>
      <td>v0.10.8</td>
      <td>Required</td>
      <td>Default value</td>
    </tr>
    <tr>
      <td>SfHeader</td>
      <td>-</td>
      <td>logoHeight</td>
      <td>false</td>
      <td>35</td>
    </tr>
    <tr>
      <td>SfHeader</td>
      <td>-</td>
      <td>logoWidth</td>
      <td>false</td>
      <td>34</td>
    </tr>
  </tbody>
</table>
```

Reply:
I saw this in the previous release, but didn't fully understand the purpose. I'll work on adding this in and understanding what's going on here.